### DISCUSSION TITLE
Validity of Supervised UMAP as Preprocessing Step to Supervised Classifier

### DISCUSSION THREAD

**Discussion by alexk101** [lmcinnes/umap#1047] [Q&A]
Hello,

I am quite a fan of UMAP and have been applying it to a number of ongoing projects as a dimensionality reduction step for very large datasets. My question is quite simple, is it valid to do a supervised reduction of the data using UMAP and then feed that data into another supervised model (in our case LightGBM)? I am worried that doing so would constitute leaking of the labels into the training data.

Thanks
